src/deadline/nuke_submitter/ui/components/scene_settings_tab.py

```diff
@@ -89,9 +201,27 @@ def refresh_ui(self, settings: RenderSubmitterUISettings):
 
         self.proxy_mode_check.setChecked(settings.is_proxy_mode)
 
+        self.timeout_checkbox.setChecked(settings.timeouts_enabled)
+
+        def _set_timeout(
+            timeout_boxes: tuple[QLabel, QSpinBox, QSpinBox, QSpinBox], timeout_seconds: int
+        ):
+            days = timeout_seconds // 86400
+            hours = (timeout_seconds % 86400) // 3600
+            minutes = (timeout_seconds % 3600) // 60
```

Comment:
Need to remove the days as well as the hours
```suggestion
            minutes = (timeout_seconds % 86400 % 3600) // 60
```

Reply:
Oops nevermind, that still works.